awscli/customizations/s3/subcommands.py

```diff
@@ -605,6 +605,36 @@ def _get_bucket_name(self, path):
         return path
 
 
+class PresignCommand(S3Command):
+    NAME = 'presign'
+    DESCRIPTION = ("Generate a pre-signed URL for an Amazon S3 object. "
+                   "This allows anyone who receives the pre-signed URL "
+                   "to retrieve the S3 object with an HTTP GET request.")
+    USAGE = "<S3Uri>"
+    ARG_TABLE = [{'name': 'path',
+                  'positional_arg': True, 'synopsis': USAGE},
+                 {'name': 'expires-in', 'nargs': '?',
```

Comment:
Is ``'?'`` is intentional for ```nargs`` for ``expires-in``? Seems strange that you can provide ``--expires-in`` without a value especially since ``default`` is already being set.